README.rst

```diff
@@ -87,56 +94,31 @@ Create a test file in the tests folder
         def empty_method():
             pass
 
-Create a Dockerfile-splunk file
-
-.. code:: Dockerfile
-
-    ARG SPLUNK_VERSION=latest
-    FROM splunk/splunk:$SPLUNK_VERSION
-    ARG SPLUNK_APP=TA_UNKNOWN
-    ARG SPLUNK_APP_PACKAGE=package
-    COPY deps/apps /opt/splunk/etc/apps/
-
-    COPY $SPLUNK_APP_PACKAGE /opt/splunk/etc/apps/$SPLUNK_APP
-
-
-Create a docker-compose.yml update the value of SPLUNK_APP
-
-.. code:: yaml
-
-    version: "3.7"
-    services:
-    splunk:
-        build:
-        context: .
-        dockerfile: Dockerfile-splunk
-        args:
-            - SPLUNK_APP=xxxxxxx
-        ports:
-        - "8000"
-        - "8089"
-        environment:
-        - SPLUNK_PASSWORD=Changed@11
-        - SPLUNK_START_ARGS=--accept-license
-
 Run pytest with the add-on and SA-eventgen installed and enabled in an external Splunk deployment
 
-.. code::: bash
+.. code:: python3
 
         pytest \
         --splunk-type=external \
-        --splunk-type=external \
         --splunk-host=splunk \
         --splunk-port=8089 \
         --splunk-password=Changed@11 \
         -v
 
-Run pytest with the add-on and SA-eventgen installed and enabled in docker
+Run pytest with the add-on and SA-eventgen installed and enabled in kubernetes
+
+Deploy `splunk-operator at cluster-scope in kubernetes cluster`_.
+
+Generate addon SPL with `ucc-gen`_.
```

Comment:
`ucc-gen` does not generate SPL, can we please be more specific how to generate SPL?

Reply:
It is not, you are correct, but you require spl to work with PSA, so I think it makes sense to give an instruction how to create one. If it is `ucc-gen` + `slim` + rename file to spl, that's fine as well.

Reply:
It would not be appropriate to jot down spl generation steps in PSA docs, as there could be updates for UCC in the future and the steps written right now could very well be useless and users might face problem. IMO we should just provide a reference to the steps for SPL creation in UCC repo, that way we do not need to keep track of updates in UCC repo also the addon in concern could be a third party addon ( not developed by splunk) and not use UCC (ucc-gen) at all so the user would need to generate the build as per the addon's requirements. This was about creating a build now about packaging we can provide a reference to splunk-packaging-toolkit and ask user to use that but again this is not something we want user to be, we just need an spl file that is installable on Splunk.

The problem with provided SPL generating steps is, if any other module (ucc-gen or slim ) does not work as user expects it to it would be considered as an issue on PSA which is not the case, user can generate the SPL using any tools to his/her disposal.